bin/zkCli.cmd

```diff
@@ -17,7 +17,7 @@ REM limitations under the License.
 setlocal
 call "%~dp0zkEnv.cmd"
 
-ZOO_LOG_FILE=zookeeper-%USERNAME%-cli-%COMPUTERNAME%.log
+set ZOO_LOG_FILE=zookeeper-%USERNAME%-cli-%COMPUTERNAME%.log
 
 set ZOOMAIN=org.apache.zookeeper.ZooKeeperMain
```

Comment:
@wordfeng
- Yes, this bug has been fixed by [ZOOKEEPER-3453](https://issues.apache.org/jira/browse/ZOOKEEPER-3453)
- Please change this github description to "`ZOOKEEPER-3453: missing 'SET' in zkCli on windows`"

Reply:
ok, i changed it